projects/core/src/http/http-timeout/default-http-timeout.config.ts

```diff
@@ -9,8 +9,7 @@ import { OccConfig } from '../../occ/config/occ-config';
 export const defaultBackendHttpTimeoutConfig: OccConfig = {
   backend: {
     timeout: {
-      // [CXSPA-2234] (breaking change): uncomment it only in the next major (v6.0?):
-      // server: 10_000,
+      server: 10_000,
```

Comment:
shouldn't it be 20_000 for 20s

Reply:
I think so ;)

Reply:
😅 Initially I thought 10s, but then decided to be more generous with 20s. However it seems I didn’t commit my change. 🤦 
Now pushed update 20_000.